ludwig/datasets/titanic/config.yaml

```diff
@@ -0,0 +1,7 @@
+version: 1.0
+split_filenames:
+  train_file: titanic_train.csv
+  test_file: titanic_test.csv
+download_file_type: csv
+csv_filename: titanic.csv
+download_urls: ["http://fake.org", "http://fake1.org"]
```

Comment:
Maybe here we can just put `None`

Reply:
Actually I was thinking of just removing this altogether as there is not really a concept of a download url since the kaggle client abstracts that away from ys

Reply:
Yeah, looks like we can remove it and replace it with:

```
competition: titanic
```